WebContent/WEB-INF/dox/en/pointValueLogging.htm

```diff
@@ -59,12 +59,22 @@
 </p>
 
 <p>
-  The <b>Purge</b> settings determine how long a point's historical values should be kept in the database. Values can 
+  The <b>Purge strategy</b> option allows user to choose strategy of data purging. Two types are supported: <b>Period</b> and <b>Values Limit</b>
```

Comment:
"The purge process runs automatically every morning at 3:05am local time."  
I would suggest moving from paragraph "Period".